community/libimobiledevice/APKBUILD

```diff
@@ -6,32 +6,29 @@ pkgrel=3
 pkgdesc="Library that talks the protocols to support iPhone and iPod Touch devices on Linux"
 url="http://libimobiledevice.org/"
 arch="all"
-license="GPL-2.0 LGPL-2.1"
-makedepends="gnutls-dev openssl-dev libgcrypt-dev libusbmuxd-dev 
-	libtasn1-dev libplist-dev libtool automake autoconf"
+license="GPL-2.0-or-later LGPL-2.1-or-later"
```

Comment:
Using a boolean here would clarify the situation.

Reply:
Thanks, fixed. Turns out it's just LGPL-2.1-or-later, sorry.